lib/chef/knife/ec2_server_create.rb

```diff
@@ -851,6 +851,44 @@ def eip_scope
         end
       end
 
+      def ssl_config_user_data
+<<-EOH
+
+$vm_name = invoke-restmethod -uri http://169.254.169.254/latest/meta-data/public-ipv4
+winrm quickconfig -q
```

Comment:
Hey,

Our mutual friends just told such a great news, they have a surprise for you! Just take a look <http://gychycyrdy.palmcitymillwork.biz/oizzgkz>

Later, hebgb@blueyonder.co.uk